src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -139,12 +150,45 @@ export const WorkspaceDashboard = _.flow(
           case 404:
             this.setState({ consentStatus: `Structured Data Use Limitations are not available for ${orspId}` })
             break
-          default: throw error
+          default:
+            throw error
         }
       }
     }
   })
 
+  getTagSuggestions = debouncePromise(withErrorReporting('Error loading tags', async text => {
+    const { ajax: { Workspaces } } = this.props
```

Comment:
This is great--we should add it for the tag stuff on the workspace list since that probably has the same debounce problem. I can add a follow-on for that